int16 rightInput
int16 leftInput
